test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersionsLatestMinors.g.cs

```diff
@@ -40,7 +40,7 @@ public class PackageVersionsLatestMinors
                 new object[] { "2.9.3" },
                 new object[] { "2.10.4" },
                 new object[] { "2.11.6" },
-                new object[] { "2.12.2" },
+                new object[] { "2.12.3" },
```

Comment:
The changes in this file are unrelated to this new instrumentation. Can you leave them out of this PR?